src/modules/tasksCreator.js

```diff
@@ -0,0 +1,64 @@
+const dynamicSort = (property) => {
+  const sortOrder = 1;
+  // if(property[0] === "-") {
+  //   sortOrder = -1;
+  //   property = property.substr(1);
+  // }
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.

Reply:
Done thanks for catching this!